scripts/forge-scripts/SharedGovernorConstants.sol

```diff
@@ -0,0 +1,95 @@
+// SPDX-License-Identifier: AGPL-3.0-only
+// slither-disable-start reentrancy-benign
```

Comment:
nit: why is this line in all files? i'm worried it's heavy-handed and at worst could suppress a real reentrancy bug